testing/src/test/java/io/stargate/it/http/docsapi/DocumentApiV2TestDisable2i.java

```diff
@@ -0,0 +1,2808 @@
+package io.stargate.it.http.docsapi;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.config.DefaultDriverOption;
+import com.datastax.oss.driver.api.core.config.DriverConfigLoader;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import com.fasterxml.jackson.databind.node.TextNode;
+import com.google.common.collect.ImmutableList;
+import io.stargate.auth.model.AuthTokenResponse;
+import io.stargate.it.BaseOsgiIntegrationTest;
+import io.stargate.it.http.RestUtils;
+import io.stargate.it.http.models.Credentials;
+import io.stargate.it.storage.StargateConnectionInfo;
+import io.stargate.it.storage.StargateParameters;
+import io.stargate.it.storage.StargateSpec;
+import io.stargate.web.models.Keyspace;
+import io.stargate.web.models.ResponseWrapper;
+import java.io.IOException;
+import java.net.InetSocketAddress;
+import java.net.URLEncoder;
+import java.time.Duration;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+import net.jcip.annotations.NotThreadSafe;
+import okhttp3.MediaType;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+import okhttp3.ResponseBody;
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+@NotThreadSafe
+@StargateSpec(parametersCustomizer = "disable2i")
+public class DocumentApiV2TestDisable2i extends BaseOsgiIntegrationTest {
```

Comment:
We could keep actual test code in an abstract base class and have two concrete sub-classes one for default 2i support and the other using `parametersCustomizer`